line-bot-api-client/src/main/java/com/linecorp/bot/client/RetryableLineMessagingClientBuilder.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.bot.client;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.linecorp.bot.model.objectmapper.ModelObjectMapper;
+
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+import lombok.experimental.PackagePrivate;
+import okhttp3.Interceptor;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import okhttp3.logging.HttpLoggingInterceptor.Level;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+
+@ToString
+@Accessors(fluent = true)
+public class RetryableLineMessagingClientBuilder {
+    private static final ObjectMapper objectMapper = ModelObjectMapper.createNewObjectMapper();
+
+    /**
+     * Use {@link LineMessagingClient#builder(String)} to create instance.
+     *
+     * @see RetryableLineMessagingClient#builder(String)
+     * @see RetryableLineMessagingClient#builder(ChannelTokenSupplier)
+     */
+    @PackagePrivate
+    RetryableLineMessagingClientBuilder() {
+    }
+
+    /**
+     * API Endpoint.
+     *
+     * <p>Default value = "https://api.line.me/".
+     */
+    private URI apiEndPoint = LineClientConstants.DEFAULT_API_END_POINT;
+
+    /**
+     * API Endpoint.
+     *
+     * @deprecated use {@link #apiEndPoint(URI apiEndPoint)}.
```

Comment:
Since this is new API, how about remove this?

Reply:
thanks, applied.